underscore.js

```diff
@@ -1105,8 +1104,22 @@
     return _.pick(obj, iteratee, context);
   });
 
-  // Fill in a given object with default properties.
-  _.defaults = createAssigner(_.allKeys, true);
+  // Fill in an object with default properties.
+  _.defaults = function() {
+    var dest = {};
+    for (var index = arguments.length - 1; index >= 0; index--) {
```

Comment:
That's the intention, since I can't test for presence on the `destination` object (that was the old `destination[prop] === void 0`). This way, we can ensure all the proper values are set, using only the `source`'s `value` (`value !== void 0`) during the iteration.